account_invoice_crm_tag/models/sale_order.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2023 Tecnativa - Stefan Ungureanu
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    @api.depends("order_line.invoice_lines")
+    def _get_invoiced(self):
+        res = super()._get_invoiced()
+        self.invoice_ids.tags_ids = self.tag_ids
+        return res
```

Comment:
This is not correct, you have to search the method used to create the invoices from the sale.order and see how it feels the info.

Reply:
It's a `_prepare...` method in `sale.order.line` and you have to use `[(4, id)]` syntax.

Reply:
I saw both methods from `sale.order` and `sale.order.line`, but I wasn't sure which to use. Finally i decided to use the `sale.order` one, but still wondering what is the best decision.